### PR TITLE
Refactoring code, add location and added twitter and facebook in the StationDTO.

### PR DESCRIPTION
**Pull request by pablogrela**
Refactoring code, add location and added twitter and facebook in the StationDTO.
 - Code Formatting
 - Remove deprecated methods
 - Change if`s to switch
 - Add location intent to google maps
- The facebook and twitter url is added to the StationDTO, that is to say the information is recieved by the       request https://cuacfm.org/radioco/api/1/radiocom/radiostation.
- The text non elements textView has been changed to make it look better.
- Change the String baseURL to https://cuacfm.org/radioco/api/1/radiocom/

### CODE REVIEW

File: app/src/main/java/justforcommunity/radiocom/activities/Home.java [hunk 1]
```diff
@@ -302,27 +310,38 @@ public void charguePodcast() {
         processFragment(podcastFragment, mContext.getString(R.string.action_podcast));
     }
 
+    public void chargueMap() {
```
Comment (ficiverson) [inline app/src/main/java/justforcommunity/radiocom/activities/Home.java]: This method must be named loadMap

File: app/src/main/java/justforcommunity/radiocom/activities/Home.java [hunk 2]
```diff
@@ -60,13 +63,15 @@
...
 
+
     private Context mContext;
     private StationDTO station;
     private Boolean playing = false;
     private NavigationView navigationView;
     private SharedPreferences prefs;
     private SharedPreferences.Editor edit;
     private static final int REQUEST_WRITE_STORAGE = 112;
+    private static final int MY_PERMISSIONS_REQUEST_READ_CONTACTS = 112;
```
Comment (ficiverson) [inline app/src/main/java/justforcommunity/radiocom/activities/Home.java]: Why do you need to request read contacts?

File: app/src/main/java/justforcommunity/radiocom/activities/Home.java [hunk 3]
```diff
@@ -302,27 +310,38 @@ public void charguePodcast() {
...
+            Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("twitter://user?screen_name=" + station.getTwitter_user()));
             intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
             startActivity(intent);
         } catch (Exception e) {
-            ProcessBuilder("https://twitter.com/" + GlobalValues.twitterName);
+            ProcessBuilder(station.getTwitter_url());
         }
     }
 
     public void chargueFacebook() {
```
Comment (ficiverson) [inline app/src/main/java/justforcommunity/radiocom/activities/Home.java]: Rename this method to loadFacebook

File: app/src/main/java/justforcommunity/radiocom/activities/Home.java [hunk 4]
```diff
@@ -302,27 +310,38 @@ public void charguePodcast() {
...
+        String query = station.getLatitude() + "," + station.getLongitude() + "(" + label + ")";
+        String encodedQuery = Uri.encode(query);
+        String uriString = uriBegin + "?q=" + encodedQuery + "&z=16";
+        Uri uri = Uri.parse(uriString);
+        Intent mapIntent = new Intent(android.content.Intent.ACTION_VIEW, uri);
+        startActivity(mapIntent);
+    }
+
 
     public void chargueTwitter() {
```
Comment (ficiverson) [inline app/src/main/java/justforcommunity/radiocom/activities/Home.java]: Rename this method to loadTwitter

File: app/src/main/java/justforcommunity/radiocom/activities/Home.java [hunk 5]
```diff
@@ -234,7 +230,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
         // Handle action bar item clicks here. The action bar will
         // automatically handle clicks on the Home/Up button, so long
         // as you specify a parent activity in AndroidManifest.xml.
-        int id = item.getItemId();
+        //int id = item.getItemId();
```
Comment (ficiverson) [inline app/src/main/java/justforcommunity/radiocom/activities/Home.java]: remove the line if its not used

**Review by ficiverson — COMMENTED** [review]
addDrawerListener

**Comment by ficiverson** [comment]
Please make also the following changes:

-Translate all spanish comment to english
-Rename the following methods like this:
chargueEmisora->loadStation
chargueGaleria->loadGallery
chargueEmsision->loadNews
charguePodcast->loadPodcast
chargueMap->loadMap
chargueTwitter->loadTwitter
chargueFacebook->loadFacebook
ProcessBuilder->processBuilder
chargueDeveloper->loadDeveloperInfo

It was a great job!! After this minor changes I will merge PR

File: app/src/main/java/justforcommunity/radiocom/activities/Home.java (hunk 2 above)
Comment (pablogrela) [reply app/src/main/java/justforcommunity/radiocom/activities/Home.java]: For nothing.
I was testing the permissions issue for the mapFragment but in the end I did not use it and that variable escaped me.
I delete it.

File: app/src/main/java/justforcommunity/radiocom/activities/Home.java (hunk 5 above)
Comment (pablogrela) [reply app/src/main/java/justforcommunity/radiocom/activities/Home.java]: ok

**Comment by pablogrela** [comment]
Done

**Pull request merged**